### ISSUE TITLE
Validation errors when opening azure-pipelines.yml with Common Language Editor Supporting TextMate Bundles

### ISSUE THREAD

**Issue by stevenvolckaert** [madskristensen/TextmateBundleInstaller#568]
Hi @madskristensen!

I don't have the TextmateBundleInstaller extension installed, however I did install Web Essentials for Visual Studio 2019.

Since the update to Visual Studio 16.8.2 yesterday I get the following error on my `azure-pipelines.yml` file in the root of my repo. The pipeline definition works perfectly in Azure DevOps.

Error 768 Unable to load schema from 'https://raw.githubusercontent.com/microsoft/azure-pipelines-vscode/v1.174.2/service-schema.json': unable to get local issuer certificate.

I did some Googling and it appears to come from the TextmateBundleInstaller? However, the issue also appears when I open my yml file with **Source Code (Text) Editor**.

How can I disable or configure the YAML validation against the schema?

Many thanks!!

Steven

**Comment by madskristensen** [comment]
No, this comes from the new YAML editor finding issues in the JSON schema file associated with the Azure Pipelines. Textmate is not involved here at all. Let's ask @sayedihashimi who might have an idea?

**Comment by sayedihashimi** [comment]
@stevenvolckaert can you report a problem https://docs.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2019 so that the team can take a look? If you provide the url of the new issue here I can route it to my team.

**Comment by irontoby** [comment]
The issue was reported here and is marked "Under Consideration":

https://developercommunity.visualstudio.com/content/problem/1276696/validation-errors-when-opening-azure-pipelinesyml.html

Is there any way to disable these errors in the meantime? Do I need to uninstall all the Azure components in VS? Can I somehow mark this file so that it's not analyzed?

I have `azure-pipelines.yml` included in several solutions and these false errors make it a lot more difficult to spot real issues.
